<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(54, 54, 54);color: rgb(177, 177, 177);
alternate-background-color: rgb(97, 97, 97);
selection-background-color: rgb(97, 97, 97);
</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>530</width>
         <height>130</height>
        </size>
       </property>
       <property name="title">
        <string>Camera settings</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="camera_box">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(54, 54, 54);color:rgb(173, 173, 173)</string>
        </property>
       </widget>
       <widget class="QComboBox" name="resolution">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(54, 54, 54);color:rgb(173, 173, 173)</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>530</width>
         <height>130</height>
        </size>
       </property>
       <property name="title">
        <string>Comport settings</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Port command:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="port_command">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="port_name">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>COM port name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>30</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>BaudRate:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="port_boundrate">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>30</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>70</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ByteSize:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="port_bytesize">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>70</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>530</width>
         <height>130</height>
        </size>
       </property>
       <property name="title">
        <string>Files settings</string>
       </property>
       <widget class="QLineEdit" name="dir_path">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="choose_dir_btn">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 173, 173); font-size: 10pt;color:rgb(54, 54, 54)</string>
        </property>
        <property name="text">
         <string>Choose dir</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt;</string>
        </property>
        <property name="text">
         <string>Pic save path:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="delay">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt;</string>
        </property>
        <property name="text">
         <string>Pre delay (sec):</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(173, 173, 173);color:rgb(54, 54, 54)</string>
       </property>
       <property name="locale">
        <locale language="English" country="TrinidadAndTobago"/>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
